Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -26,6 +26,8 @@ public class NavigationPage : Page, IPageContainer<Page>, IBarElement, INavigati
 
 		public static readonly BindableProperty BarTextColorProperty = BarElement.BarTextColorProperty;
 
+		public static readonly BindableProperty IconColorProperty = BindableProperty.Create(nameof(IconColor), typeof(Color), typeof(NavigationPage), Color.Default);
```

Comment:
why is TitleIcon attached (and applied to the Page inside of a NavPage) and IconColor a normal BP, applied to the NavPage ?

Reply:
@StephaneDelcroix nice point
i will update this PR with attached property